user_guide_src/source/database/queries/030.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+    $data = [
+        'id'          => new RawSql('DEFAULT'),
+        'title'       => 'My title',
+        'name'        => 'My Name',
+        'date'        => '2022-01-01',
+        'last_update' => new RawSql('CURRENT_TIMESTAMP()'),
+    ];
+
+    $builder->insert($data);
+    /* Produces:
+        INSERT INTO mytable (id, title, name, date, last_update)
+        VALUES (DEFAULT, 'My title', 'My name', '2022-01-01', CURRENT_TIMESTAMP())
+    */
```

Comment:
Remove this sample.
This sample code is weird. Because it is the explanation for `$db->escape()` but there is no `$db->escape()`.

Reply:
I added escape() test sample instead.